compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/CreateViewTransform.java

```diff
@@ -112,20 +113,12 @@ public void onWatermark(final Watermark inputWatermark) {
   @Override
   public void close() {
     onWatermark(new Watermark(BoundedWindow.TIMESTAMP_MAX_VALUE.getMillis()));
-
-    if (!isEmitted) {
```

Comment:
I think so (PerPercentile test passes)
It will be handled more explicitly in https://github.com/apache/incubator-nemo/pull/159/files#diff-30a095310e20d5a90938099d206b1757R32